doc/deployment_guide.md

```diff
@@ -88,22 +64,88 @@ Red Hat®, Red Hat Enterprise Linux®, and Ceph are trademarks or registered tra
 <div style="page-break-after: always;"></div>
 
 # Chapter 1 Overview
-This guide provides information necessary to deploy the Dell EMC Ready Architecture for Red Hat OpenStack platform v13.0 using an automation framework developed by Dell EMC and validated by Red Hat.
+This guide provides information necessary to deploy the Dell EMC Ready Architecture for Red Hat OpenStack platform v13.0 using an automation framework developed by Dell EMC & Validated by Redhat (JetPack v13.0)

```

Comment:
Why do you have (JetPack v13.0) at the end of this statement?  Also, Red Hat did NOT physically validate JetPack 13.0.  They reviewed the RA and that was all for this release.

Reply:
Thanks, am updating the document accordingly.